bundles/org.openhab.binding.juicenet/src/main/resources/OH-INF/thing/juicenet-device.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="juicenet"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
+
+		<label>JuiceBox Charger</label>
+		<description>JuiceBox EV-Charger</description>
+
+		<channels>
+			<channel id="name" typeId="nameType"/>
```

Comment:
General: The channel type definitions can have the same name as the channels, so the "Type" postfix is not needed.